docs/topics/using-openrewrite-recipes.adoc

```diff
@@ -53,7 +53,7 @@ You can include more than one recipe by specifying each in the `activeRecipes` p
 +
 [source,terminal,subs="attributes+"]
 ----
-$ ./{mta-cli} transform openrewrite --input </path/to/source/project> \
+$ ./{mta-cli} --openrewrite --input </path/to/source/project> \
```

Comment:
This should be `./{mta-cli} transform openrewrite --input </path/to/source/project> \`

Reply:
Fixed in PR- https://github.com/windup/windup-documentation/pull/828